### ISSUE TITLE
.tar.bz2 download link broken

### ISSUE THREAD

**Issue by cig0** [podenv/podenv#97]
Hi,

The download link is broken

```shell
curl -L https://github.com/podenv/podenv/releases/download/0.1.0/podenv-x86_64-linux.tar.bz2 -o -                   
Not Found
```

Thanks!

**Comment by TristanCacqueray** [comment]
Hello, the release is not available yet. I'll publish the current HEAD soon, thanks for the feedback.

**Comment by TristanCacqueray** [comment]
The first release is now published (but the link changed from `0.1.0` to `v0.1.0`):
    https://github.com/podenv/podenv/releases/download/v0.1.0/podenv-x86_64-linux.tar.bz2
